### PR TITLE
* PySTAC notebook

### PR DESCRIPTION
**Pull request by dustgalactic**

### CODE REVIEW

**Review by andher1802 — APPROVED** [review]
LGTM! We can discuss and update the authentication issue after publishing.

File: README.md [hunk 1]
```diff
@@ -110,6 +110,20 @@ order data.
...
+
+SpatioTemporal Asset Catalog (STAC) is a specification designed to establish
+a standard in geospatial data. UP42 has a STAC API set `/v2/assets/stac` that allows
+you to retrieve STAC objects mapped to your UP42 assets.
+
+### Jupyter notebook
+
+In the `notebooks` directory there is a Python
+[Jupyter](https://jupyter.org) notebook that makes use of the
+[UP42 SDK](https://sdk.up42.com/) and
```
Comment (perusio) [inline README.md:123]: It uses the SDK? Then it needs to just use the SDK and get rid of the _custom_ auth altogether.

File: stac-asset-service/notebooks/pystac-stac-asset-service.ipynb [hunk 2]
```diff
@@ -0,0 +1,1667 @@
...
+    "## Import libraries <a class=\"anchor\" id=\"import\"></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import up42\n",
```
Comment (perusio) [inline stac-asset-service/notebooks/pystac-stac-asset-service.ipynb:34]: If the SDK also makes available `pystac_client` why do you do things as if the SDK wasn't available? E.g., the auth, the downloading of assets, etc.

File: README.md (hunk 1 above)
Comment (perusio) [reply README.md:123]: Ditto for handling asset download, etc.

**Review by perusio — APPROVED** [review]
I left my comments.

**Pull request merged**